### ISSUE TITLE
is this mamba thing really required ?

### ISSUE THREAD

**Issue by Username69992872** [JonathanFly/bark#58]
Actually I just created a virtual environment and made a git + install pip requirements.
Works fine so far, also webui starts, however Voice generation only works on CPU it says. So i guess that mamba thing is for cuda. 
Automatic1111 seems to work without mamba?

**Comment by JonathanFly** [comment]
No, it's just one way to install it. If you can install Cuda and Pytorch and the libraries, you can just the requirements.txt files instead.

**Comment by gitihobo** [comment]
I did the same thing but I am getting Loading Bark models...
preload_models No GPU being used. How can I make it use gpu?

**Comment by Username69992872** [comment]
> No, it's just one way to install it. If you can install Cuda and Pytorch and the libraries, you can just the requirements.txt files instead.

thx for the reply. Can you pls give a bit more detail how to do it now exactly without miniforge and just using venv?
a 1 click installer of course would be the greatest :D

There is no pure requirements.txt:
requirements-dev-pip.txt
requirements-pip.txt
which one to use ?

I guess i have to add those from environment-cuda.yml ?

**Comment by fnrcum** [comment]
If you are using a normal python venv then activate that venv and install this
```py
pip install torch==2.0.0+cu118 torchvision --extra-index-url https://download.pytorch.org/whl/cu118
```

Note that `cu118 ` from both parts of the commands must match the cuda version you have installed

**Comment by gitihobo** [comment]
torchaudio 2.0.2 requires torch==2.0.1, but you have torch 2.0.0+cu118 which is incompatible. should I install it?

**Comment by JonathanFly** [comment]
> 

I haven't tested the pip, but the mamba install does give you torch==2.0.1 at the end, so that sounds right.

**Comment by gitihobo** [comment]
I did that but now it says the opposite ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
torchvision 0.15.1+cu118 requires torch==2.0.0, but you have torch 2.0.1 which is incompatible.

**Comment by JonathanFly** [comment]
I'll be in Bark Discord probably very late Monday (Tuesday AM really).
